CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 * [#1913](https://github.com/ruby-grape/grape/pull/1913): Fix multiple params validators to return correct messages for nested params - [@bikolya](https://github.com/bikolya).
 * [#1926](https://github.com/ruby-grape/grape/pull/1926): Fixes configuration within given or mounted blocks - [@myxoh](https://github.com/myxoh).
 * [#1937](https://github.com/ruby-grape/grape/pull/1937): Fix bloat in released gem - [@dblock](https://github.com/dblock).
+* [#1930](https://github.com/ruby-grape/grape/pull/1930): Move block call to separate method so it can be spied on - [@estolfo](https://github.com/estolfo).
```

Comment:
This version was shipped, changelog has to move to Next, please.

Reply:
ok done 👍🏼